projects/canopy/src/lib/carousel/carousel.component.html

```diff
@@ -2,6 +2,31 @@
   <lg-heading [level]="headingLevel" class="lg-carousel__heading lg-visually-hidden">{{
     description
   }}</lg-heading>
+
+  <div
+    *ngIf="autoPlay"
+    aria-live="polite"
+    aria-atomic="true"
+    class="lg-carousel__autoplay"
```

Comment:
I'm wondering if this autoplay should be in its own component?

I would also change the classes to use a single element. For example I would have:
```
lg-carousel-autoplay
lg-carousel-autoplay__btn
lg-carousel-autoplay__btn--pause
lg-carousel-autoplay__btn--play
```

Reply:
yes, moved it in component and updated the class names.